### ISSUE TITLE
Aishell

### ISSUE THREAD

**Issue by tongjinle123** [gentaiscool/end2end-asr-pytorch#5]
we tried aishell1-dataset using custom dataset method and reported to have a bad result. 
"""
python train.py --cuda --train-manifest data_aishell/aishell1.csv --device-ids 1 --val-manifest data_aishell/aishell1_dev.csv --test-manifest data_aishell/aishell1_test.csv --labels-path data/labels/aishell_labels.json --emb_cnn --shuffle --epochs 1000 --batch-size 32
"""
in epoch 50 : CER -train :48 CER -valid:85, WER 100.
the CER -valid remains 80+ for the whole 50-epoch.
is there any idea that probably result in the mistake? such as we need to have 1000 epoch to have a better result? or we should build a new labels.json for the aishell1 ---we use the original aishell_label.json in the project.

**Comment by gentaiscool** [comment]
hi @tongjinle123, you should build your own labels.json before you start training otherwise the characters are unknown. 

Let me know if you have problems in training the model. 
Thanks

**Comment by gentaiscool** [comment]
Sorry for late reply. I tried to run with smaller models (2 layers) and achieved ~20% CER (after 400 epochs). I think you need to tune the parameters, and run for more epochs. I will put the results in README.md once I finished testing the model. Thank you
